lib/galaxy/model/__init__.py

```diff
@@ -1280,6 +1261,10 @@ def disk_size( cls ):
         # label creates a scalar
         return size_query.label( 'disk_size' )
 
+    def disk_nice_size( self ):
```

Comment:
It may make sense to decorate this method with `@property` and call it without the parentheses.